assignment/index.html

```diff
@@ -62,21 +62,59 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
+      var jsonToCsv = function(json){console.log(json);};
+
+      let healthCentersInfo = [['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']];
```

Comment:
It'd be good to not have to type out each of these keys manually.

Take a look how at this: `Object.keys(healthCenters[0])` Does all the work for you!